sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/DirectoriesImpl.java

```diff
@@ -112,7 +113,8 @@ Mono<Response<Void>> createNoCustomHeaders(@HostParam("url") String url,
             @HeaderParam("x-ms-file-last-write-time") String fileLastWriteTime,
             @HeaderParam("x-ms-file-change-time") String fileChangeTime,
             @HeaderParam("x-ms-file-request-intent") ShareTokenIntent fileRequestIntent,
-            @HeaderParam("Accept") String accept, Context context);
+            @HeaderParam("x-ms-owner") String owner, @HeaderParam("x-ms-group") String group,
```

Comment:
let's keep each headerparam in separate line for better readability.

Reply:
this is generated code, I am unable to change anything in this file unfortunately

Reply:
I might have missed that this was part of the AutoRest-generated code. You can ignore this.